test/e2e.js

```diff
@@ -525,33 +504,25 @@ describe('plotly database connector', function Spec() {
             expect(error.toString()).to.contain('NoSuchElementError');
         });
 
-        it('should show a log with a new logged item',
-        async () => {
-            const expectedClass = 'test-5-entries';
-            const logs = await this.getLogs();
-
-            const testClass = await this.getClassOf(logs);
-            expect(testClass).to.contain(expectedClass);
-        });
-
         after(close);
 
     });
 
     describe('-> the API ', () => {
-        const testedDialect = DIALECTS.MYSQL;
 
         before(openApp);
 
-        describe('/connect', () => {
+        const testedDialect = dialectUnderTest;
+
+        describe('/v0/connect', () => {
             it('if app is "initialized", returns an error and no databases',
             async() => {
                 const expectedClass = `test-${APP_STATUS.INITIALIZED}`;
                 const btn = await this.getConnectBtn();
                 const testClass = await this.getClassOf(btn);
                 expect(testClass).to.contain(expectedClass);
 
-                await fetch(`http://${BASE_URL}/connect`)
+                await fetch(`http://${BASE_URL}/v0/connect`)
```

Comment:
I think we should drop `v0` support from our code and our tests since our front-end managed to not have to use it. Otherwise, we're spending time maintaining something that we're not actually really using. It's great for us to have the code patterns to add additional API URLs though, so it wasn't totally for loss!